### ISSUE TITLE
Documentation of the scripts option is misleading

### ISSUE THREAD

**Issue by rafaelbco** [plone/plone.recipe.zope2instance#29]
The docs may lead to the impression that the user must include the eggs' names in the `scripts` option, in order to generate scripts for those eggs.

I found out this is wrong. What you need to do is add the _scripts' names_ to the option AND also make sure the corresponding eggs are listed _explicitly_ on the `eggs` option, even if an egg is a dependency of an already listed egg.

**Comment by rafaelbco** [comment]
Suggestion of text:

---
scripts

Add this parameter with no arguments to suppress script generation. Otherwise (i.e. without this parameter), scripts for packages added to the eggs parameter will be generated. You may also configure per script. E.g.:

```
[instance]
recipe = plone.recipe.zope2instance
eggs =
  Plone
  mr.migrator
  my.package
scripts = my_package_script
```

In the above example, only the `my_package_script` will be generated. Keep in mind that the egg containing the script (`my.package` in the example) must be listed explicitly in the `eggs` option, even if it's a dependency of an already listed egg.
---

**Comment by gforcada** [comment]
Would you mind creating a pull request? That will be far more easy to integrate 😃

**Comment by rafaelbco** [comment]
No problem. When I have the time ...

I like to fill issue reports quickly so I don't let the issue pass and
later submit a PR.


Rafael Oliveira

**Comment by dataflake** [comment]
This was fixed by e9807380a1561d36786da00d78dca5edf883eba9
